src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.Development.json

```diff
@@ -4,7 +4,7 @@
     "PublicStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://data-storage:10001/devstoreaccount1;",
     "NotificationStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://data-storage:10001/devstoreaccount1;",
     "ContentDb": "Server=db;Database=content;User=SA;Password=Your_Password123;Trusted_Connection=false;",
-    "StatisticsDb": "Server=db;Database=master;User=SA;Password=Your_Password123;"
+    "StatisticsDb": "Server=db;Database=public-statistics;User=SA;Password=Your_Password123;"
```

Comment:
We don't have a public-statistics database in Docker, at least not yet anyway so this isn't going to work